bentoml/_internal/configuration/containers.py

```diff
@@ -24,7 +24,7 @@
 if TYPE_CHECKING:
     from multiprocessing.synchronize import Lock as SyncLock  # noqa: F401
 
-    from .. import ext_typing as ext
+    from .. import external_typing as ext  # noqa # pylint: disable=unused-import
```

Comment:
why do we need `disable=unused-import` here?

Reply:
does the next line that imports `PrometheusClient` need it as well? is there a way to disable this check for TYPE_CHECKING related imports?

Reply:
pep8 was complaining about this

Reply:
I think we already disable this for `TYPE_CHECKING`, but the pep8 bot still seem to complain

Reply:
It's an example of a false positive raised by flake8. That's why we abandon flake8. It has many overlaps with pylint, and many differences are false positives.
We are not using flake8 as the linter anymore. So we can just remove those #noqa #pylint things.

> is there a way to disable this check for TYPE_CHECKING related imports?
That's the behavior of pylint now.

We can turn off the flake8 checking in our editors now.